.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file by adding a `.gitignore` file and paste the following code

```
.vscode
```

This will inform git not to track files that are related to your visual studio code settings.